### PR TITLE
Modify historical returns to allow compounding

### LINKED ISSUE #85: Compounding in `mean_historical_returns`
Hi Robert, I'm new to portfolio management and have been using PyPortfolioOpt for the past few days. I have been finding the library extremely useful and interesting.

I was hoping you could clarify the way `mean_historical_returns` annualises returns, according to [this](https://github.com/robertmartin8/PyPortfolioOpt/blob/7a06bee5efc0c061b41485c0c4b1e74e38eff9a5/pypfopt/expected_returns.py#L70) line the mean daily returns are multiplied by 252 (the number of trading days in a year). Shouldn't the mean value be compounded over this period? 

Thus if the mean daily return for an equity is `x`, the annualised mean daily return would be `[ (1+x) ^ 252 ] - 1` instead of `x * 252`. Would be more than happy to submit a PR if indeed this should be changed.

### PR DESCRIPTION
**Pull request by gauravsofat**
Hi @robertmartin8,

This PR resolves #85 
I have manually tested the code comprehensively with a wide range of values but have not been able to use `pytest` for the same because of some encoding errors that kept popping up on my system. 
Since this was a relatively small fix, I have made the PR on the basis of this manual testing itself.

### CODE REVIEW

File: pypfopt/expected_returns.py [hunk 1]
```diff
@@ -66,8 +66,13 @@ def mean_historical_return(prices, frequency=252):
     if not isinstance(prices, pd.DataFrame):
         warnings.warn("prices are not in a dataframe", RuntimeWarning)
         prices = pd.DataFrame(prices)
-    returns = returns_from_prices(prices)
-    return returns.mean() * frequency
+    meanReturns = returns_from_prices(prices).mean()
```
Comment (schneiderfelipe) [inline pypfopt/expected_returns.py:69]: I think we are better off keeping the already used variable name (`returns`) or, at least, comply with [PEP8](https://www.python.org/dev/peps/pep-0008/#function-and-variable-names) and don't use camel case (`mean_returns` instead).

File: pypfopt/expected_returns.py [hunk 2]
```diff
@@ -66,8 +66,13 @@ def mean_historical_return(prices, frequency=252):
...
+    if compounded:
+        meanReturns = meanReturns.apply(lambda x: pow(1 + x, frequency) - 1)
```
Comment (schneiderfelipe) [inline pypfopt/expected_returns.py:71]: I think it is important to add a (simple at least) test for this.

**Comment by robertmartin8** [comment]
@gauravsofat 

Thanks for the initial PR! I am going to need to test your code on my local machine. In particular, I am interested to know how the performance of your lambda compares with direct operation on the pandas series

```python
result = (1+returns.mean())**frequency
```

I will figure it out, add a test, and push it along with a bunch of other changes that I am making. Will credit you somewhere in the docs :)

**Comment by gauravsofat** [comment]
Thanks guys, I completely agree with your suggestions. Closing this PR in accordance with the same.

**Pull request closed without merging**